web/src/components/TestSpecForm/SelectorSuggestions.tsx

```diff
@@ -0,0 +1,42 @@
+import {ArrowLeftOutlined, InfoCircleOutlined} from '@ant-design/icons';
+import {Tooltip} from 'antd';
+
+import {ISuggestion} from 'types/TestSpecs.types';
+import * as S from './TestSpecForm.styled';
+
+interface IProps {
+  onClick(query: string): void;
+  onClickPrevSelector(query: string): void;
+  prevSelector: string;
+  selectorSuggestions: ISuggestion[];
+}
+
+const SelectorSuggestions = ({onClick, onClickPrevSelector, prevSelector, selectorSuggestions}: IProps) => (
+  <>
+    {Boolean(selectorSuggestions.length) && (
+      <>
+        <S.FormSectionText>
+          <InfoCircleOutlined /> Try to select spans using these suggestions:
+        </S.FormSectionText>
+
+        {Boolean(prevSelector) && (
+          <Tooltip title={prevSelector}>
+            <S.SuggestionsButton onClick={() => onClickPrevSelector(prevSelector)} type="link">
+              <ArrowLeftOutlined /> Prev selector
```

Comment:
I was looking into this in the UI and I'm not sure if we just need to have it more clear as an `undo` option